docs/src/PageHeader.doc.js

```diff
@@ -0,0 +1,645 @@
+        `}
+        defaultCode={`
+<PageHeader
+  title="Settings"
+  primaryAction={
+    <Tooltip text="Additional options">
+      <IconButton icon="ellipsis" iconColor="darkGray" />
+    </Tooltip>
+  }
+/>;
+        `}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="do"
+        description={`
+        Plan for most PageHeaders to be full width. A \`maxWidth\` should only be supplied when the content of the page is center aligned.
+        `}
+        defaultCode={`
+<Flex gap={4} direction="column" flex="grow">
+  <PageHeader
+    title="Settings"
+    primaryAction={
+      <Tooltip text="Additional options" idealDirection="up">
+        <IconButton icon="ellipsis" iconColor="darkGray" />
+      </Tooltip>
+    }
+  />
+  <Box
+    display="flex"
+    marginStart={-3}
+    marginEnd={-3}
+    marginBottom={-3}
+    marginTop={-3}
+    wrap
+    width="80%"
+    direction="column"
+  >
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <Heading size="md">Edit profile</Heading>
+    </Box>
+
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <TextField
+        label="TextField 1"
+        id="textfield1"
+        onChange={() => {}}
+        placeholder="Placeholder"
+      />
+    </Box>
+
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <Box display="flex" wrap marginStart={-3} marginEnd={-3} marginBottom={-3} marginTop={-3}>
+        <Box flex="grow" paddingX={3} paddingY={3}>
+          <TextField
+            label="TextField 2"
+            id="textfield2"
+            onChange={() => {}}
+            placeholder="Placeholder"
+          />
+        </Box>
+        <Box flex="grow" paddingX={3} paddingY={3}>
+          <TextField
+            label="TextField 3"
+            id="textfield3"
+            onChange={() => {}}
+            placeholder="Placeholder"
+          />
+        </Box>
+      </Box>
+    </Box>
+
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <SelectList
+        label="SelectList"
+        id="selectlist"
+        options={[
+          {
+            value: 'belgium',
+            label: 'Belgium',
+          },
+          {
+            value: 'france',
+            label: 'France',
+          },
+          {
+            value: 'usa',
+            label: 'USA',
+          },
+        ]}
+        placeholder="Placeholder"
+        onChange={() => {}}
+      />
+    </Box>
+  </Box>
+</Flex>;
+        `}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description="Supply more than one primary style action."
+        defaultCode={`
+<PageHeader
+  title="Product groups"
+  subtext="S. E. All products USD"
+  primaryAction={<Button color="red" text="Create product group" />}
+  secondaryAction={<Button color="red" text="Promote" />}
+/>;
+        `}
+      />
+      <MainSection.Card
```

Comment:
visual bug on this one:
<img width="1062" alt="Screen Shot 2021-03-12 at 1 13 03 PM" src="https://user-images.githubusercontent.com/12059539/110999144-c4e7dd80-8334-11eb-98d5-b87ddd09cf6a.png">